packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -124,11 +129,13 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
   );
 };
 
-const getMenuItems = (
+const GetMenuItems = (
```

Comment:
As it's a component, it may be a good idea to rename it to `MenuItems` (without `get`) or something else

Reply:
Will name it MappedMenuItems